elisp/ghc-check.el

```diff
@@ -580,14 +587,21 @@ open this file and jump to the error inside it."
 (defun ghc-jump-file ()
   (interactive)
   (let* ((ovl (car (ghc-check-overlay-at 1)))
-	 (file (if ovl (overlay-get ovl 'ghc-file))))
+	 (file (if ovl (ghc-check-overlay-file ovl))))
     (if file (find-file file))))
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
-(defvar ghc-hlint-options nil "*Hlint options")
+(defcustom ghc-hlint-options nil
+  "*Hlint options"
+  :type '(string)
+  :group 'ghc-mod)
 
-(defvar ghc-check-command nil)
+(defcustom ghc-check-command nil
```

Comment:
I don't think this one should be a customize, it's set at runtime in `ghc-toggle-check-command` below. At least I don't think that makes sense, maybe a user could use this to set the default?